deploying-an-existing-application-using-okteto-ui.md

```diff
@@ -0,0 +1,175 @@
+
+Next, save the changes and commit the changes to GitHub:
+
+```bash
+(venv)$ git add okteto-stack.yaml
+(venv)$ git commit -m "Update image repository"
+(venv)$ git push
+```
+
+With the stack file updated, proceed to your Okteto dashboard to deploy the application:
+
+![Okteto Dashboard](https://res.cloudinary.com/adeshina/image/upload/v1611604440/o5fyl3fhoywmc6auigli.png)
+
+Click on the **Deploy** button. A popup will appear, fill in the **Repository URL** and optionally the branch name which in this case is *main*.
+
+![Deploy Button](https://res.cloudinary.com/adeshina/image/upload/v1611604456/wdi3swnvoa2v2bhaaqpd.png)
+
+Next, click on the deploy button. Okteto automatically retrieves, deploys and starts your application.
+
+![Deployed Application](https://res.cloudinary.com/adeshina/image/upload/v1611604494/rscsvqxobd5dc5bt0baa.png)
+
+## Adding A New Route
+
+Let's update the application by adding a route for adding bulk recipes.
+
+In `app/api.py`, just below the POST route, add another route. Start by importing the `List` wrapper from typing before proceeding to adding the new route:
+
+```py
+from typing import List
+
+...
+
+@app.post("/recipes", tags=["Recipe"])
+def add_bulk_recipe(bulk_recipe: List[RecipeSchema] = Body(...)) -> dict:
+    for recipe in bulk_recipe:
+        recipe.id = len(recipes) + 1
+        recipes.append(recipe.dict())
+
+    return {            
+        "message": "Recipes added successfully"    
+    }
+
+```
+
+Test the new route from your terminal:
+
+```bash
+(venv)$ curl --request POST \
+  --url http://0.0.0.0:8080/recipes \
+  --header 'Content-Type: application/json' \
+  --data '[
+	{
+		"name": "Donuts",
+		"ingredients": [
+			"Flour",
+			"Milk",
+			"Sugar",
+			"Vegetable Oil"
+		]
+	},
+	{
+		"name": "Creamy Donuts",
+		"ingredients": [
+			"Flour",
+			"Milk",
+			"Sugar",
+			"Vegetable Oil",
+			"Irish whipped cream"
+		]
+	}
+]'
+```
+
+You will get a response:
+
+```json
+{
+  "message": "Recipes added successfully."
+}
+```
+
+Commit the new change in the `app/api.py` file into a new branch:
+
+```bash
+(venv)$ git checkout -b bulk-recipes && git add app/api.py
+(venv)$ git commit -m "Add bulk recipes"
+(venv)$ git push -u origin bulk-recipes
+```
+
+Verify that the new branch has been created in your GitHub repo.
+
+## Redeploying
+
+Once again, login into your Okteto dashboard and click on the redeploy button:
+
+![Redeploy Button](https://res.cloudinary.com/adeshina/image/upload/v1611604547/vhenrnydsenkb6a0xd3e.jpg)
+
+Change the branch from `main` to `bulk-recipes` and deploy:
+
+![Image 5](https://res.cloudinary.com/adeshina/image/upload/v1611604639/knpe6yiz4alvplgniszb.png)
+
+The branch on which the redeployment can be seen:
+
+![bulk-recipes branch](https://res.cloudinary.com/adeshina/image/upload/v1611604587/isccfvfifunqfmnevtp8.jpg)
+
+You can go on to test the new route by replacing `localhost` from previous requests to the live application url.
```

Comment:
Add the same test as before here. I'd like the reader to make the conclusion that testing in okteto is even better than testing locally :)

Reply:
Great, I'll add that.